assets/css/style.css

```diff
@@ -51,8 +51,8 @@ h1, h2, h3, h4, h5, h6 {
   position: relative;
 }
 
-/*--------------------------------------------------------------
-# Back to top button
+/*--------------------------------------------------------------
+# Back to top button
```

Comment:
Same here.

Reply:
It's just moving lines, a bit confusing from GitHub